Analysis/Core/src/TriggerAliases.cxx

```diff
@@ -9,14 +9,15 @@
 // or submit itself to any jurisdiction.
 
 #include "AnalysisCore/TriggerAliases.h"
+#include "Framework/Logger.h"
 
-void TriggerAliases::Print()
+void TriggerAliases::AddClassIdToAlias(uint32_t aliasId, int classId)
 {
-  for (auto& al : GetAliasToClassIdsMap()) {
-    printf("%d", al.first);
-    for (auto& classIndex : al.second) {
-      printf(" %d", classIndex);
-    }
-    printf("\n");
+  if (classId < 0 || classId > 99) {
+    LOGF(fatal, "Invalid classId = %d for aliasId = %d\n", classId, aliasId);
+  } else if (classId < 50) {
+    mAliasToTriggerMask[aliasId] |= 1ull << classId;
+  } else {
+    mAliasToTriggerMaskNext50[aliasId] |= 1ull << classId;
```

Comment:
So we had 50 in the first, and 50 in the next in Run 2? Not 64 in the first?

Reply:
Yes, correct